test/odyssey/test_scram_frontend.sh

```diff
@@ -5,22 +5,22 @@ if test $PGVERSION -lt 10; then
     exit 0
 fi
 
-PGPASSWORD=incorrect_password psql -h $ODYSSEY_HOST -p $ODYSSEY_PORT -U frontend_auth_plain -c "SELECT 1" scram_db > /dev/null 2>&1 && {
-    echo "ERROR: successfully auth with incorrect password and plain password in config"
-    exit 1
-}
+#PGPASSWORD=incorrect_password psql -h $ODYSSEY_HOST -p $ODYSSEY_PORT -U frontend_auth_plain -c "SELECT 1" scram_db > /dev/null 2>&1 && {
```

Comment:
Don't we need these tests?

Reply:
Added this by mistake. I commented this for debug.